independent-projects/resteasy-reactive/server/vertx/src/test/java/org/jboss/resteasy/reactive/server/vertx/test/headers/CookiesTestResource.java

```diff
@@ -13,6 +13,7 @@ public class CookiesTestResource {
 
     @POST
     @Path("/set-cookie")
+    @SuppressWarnings("deprecation")
```

Comment:
How about using `NewCookie.Builder` instead? :)

Reply:
It would not be useful in this case as we are making a `NewCookie` out of a string, not known parts

Reply:
Ah my bad, The spec says to replace with it: `RuntimeDelegate.getInstance().createHeaderDelegate(NewCookie.class).fromString(value)`

That sucks 😜

Reply:
Yeah, it does not make much sense...